src/main/java/org/terasology/potions/events/DrinkPotionEvent.java

```diff
@@ -20,34 +20,76 @@
 import org.terasology.entitySystem.event.Event;
 import org.terasology.potions.component.PotionComponent;
 
+/**
+ * This event is sent to an entity to indicate that it'll be drinking a given potion.
```

Comment:
Treated the Potion as one.

DrinkPotionEvent sends the entire potion.
BeforeDrink (now titled BeforeApplyPotionEffectEvent) works on one potion effect at a time.